src/Components/Web3/Contracts/DBETToVETDepositContract.ts

```diff
@@ -1,44 +1,41 @@
 /* eslint-disable no-console */
 import BaseContract from './BaseContract'
 import { BigNumber } from 'bignumber.js'
-import { Subject, interval, Observable } from 'rxjs'
-import {
-    mergeMap,
-    timeout,
-    filter,
-    catchError,
-    tap,    
-} from 'rxjs/operators'
+import { Subject, Observable } from 'rxjs'
+import { timeout, filter, catchError, tap } from 'rxjs/operators'
 import { Config } from '../../Config'
 import Helper from '../../Helper'
 import Web3 from 'web3'
-import { Contract } from 'web3/types'
 import { VETDeposit } from './Deposit'
+import Contract from 'web3/eth/contract'
 
 const ethAbi = require('web3-eth-abi')
 const Contract_DBETToVETDeposit = require('../../Base/Contracts/DBETToVETDeposit.json')
-const Contract_DBETVETToken = require('../../Base/Contracts/DBETVETToken.json')
+const contracts = require('@decent-bet/contract-migration')
 
 const helper = new Helper()
 const WATCH_DEPOSIT_TIMEOUT = 9 * 60000
 export default class DBETToVETDepositContract extends BaseContract {
-    private listener: any
     private contract: Contract
     private senderContract: any
     private onProgress: Subject<any>
 
     constructor(web3: Web3, thor: Web3) {
         super(web3)
-        this.listener = null
         this.contract = new web3.eth.Contract(
             Contract_DBETToVETDeposit.abi,
             Config.depositAddress
         )
+
+        const SENDER_CONTRACT_ADDRESS =
+            Config.env === 'production'
+                ? contracts.DBETVETToken.address['0xc7']
```

Comment:
We could probably add chainTag to Config and call `Config.chainTag` here instead